ludwig/schema/trainer.py

```diff
@@ -229,7 +310,10 @@ class GBMTrainerConfig(BaseTrainerConfig):
     eval_batch_size: int = schema_utils.PositiveInteger(
```

Comment:
Metadata omitted on purpose? Is the `eval_batch_size` metadata necessarily different for GBM?

Reply:
I don't think so, though this probably means we can just use the `BaseTrainerConfig`'s `eval_batch_size` definition.